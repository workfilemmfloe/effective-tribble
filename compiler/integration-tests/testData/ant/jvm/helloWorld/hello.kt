package hello

fun main(args : Array<String>) {
    for (s in arrayList("a"))
      println("Hello, $s!")

}
