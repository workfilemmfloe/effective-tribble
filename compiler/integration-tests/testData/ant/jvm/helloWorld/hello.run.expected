OUT:
Hello, a!

Return code: 0
